modules/exploits/windows/local/cve_2020_0787_bits_arbitrary_file_move.rb

```diff
@@ -0,0 +1,242 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/post/common'
+require 'msf/core/post/windows/priv'
+require 'msf/core/post/windows/registry'
+require 'msf/core/exploit/exe'
+require 'msf/core/post/windows/filesystem'
+require 'msf/core/exploit/file_dropper'
+require 'msf/core/post/file'
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::Common
+  include Msf::Post::Windows::Priv
+  include Msf::Exploit::EXE # Needed for generate_payload_dll
+  include Msf::Post::Windows::FileSystem
+  include Msf::Post::Windows::ReflectiveDLLInjection
+  include Msf::Exploit::FileDropper
+  include Msf::Post::File
```

Comment:
`Msf::Exploit::Local` will pull in `Msf::PostMixin`, which will pull in `Msf::Post::Common`. I think the rest should be fine.

Reply:
Yep this is correct, removed that top line with next commit.